src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -78,6 +80,14 @@ public function collect(Request $request, Response $response, \Exception $except
                 $this->data['bundles'][$name] = $bundle->getPath();
             }
         }
+
+        $versionRequest = file_get_contents('http://symfony.com/roadmap.json?version='.$this->data['symfony_version']);
```

Comment:
What about keeping the result in cache and use the dates from the JSON to "dynamically" determine if the version is out of date or not? That way, we only get the HTTP hit when the version of Symfony changes or when the cache is empty.